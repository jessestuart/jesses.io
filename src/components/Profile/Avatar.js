/* @flow */
import Img from 'gatsby-image'
import React from 'react'

import classNames from 'classnames'

const Avatar = ({ image }: { image: * }) =>
  image ? (
    <Img
      className={classNames('br-100 justify-center mv2 pa0')}
      style={{
        boxShadow: '0 0 12px 12px rgba(40, 40, 40, 0.6)',
      }}
      fluid={image}
    />
  ) : null

export default Avatar
