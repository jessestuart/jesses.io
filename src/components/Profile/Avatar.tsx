import Img, { FluidObject } from 'gatsby-image'
import React from 'react'

interface Props {
  image: FluidObject
}

const Avatar = ({ image }: Props) =>
  !image ? null : (
    <Img
      className="br-100 justify-center mv2 pa0"
      loading="eager"
      style={{
        boxShadow: '0 0 12px 12px rgba(40, 40, 40, 0.6)',
      }}
      fluid={image}
    />
  )

// const Avatar = ({ image }: Props) => (
//   <StyledImg
//     className="br-100 shadow-3"
//     padding={0}
//     justifyContent="center"
//     my={2}
//     loading="eager"
//     fluid={image}
//   />
// )

export default Avatar
