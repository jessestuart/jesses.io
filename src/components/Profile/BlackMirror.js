/* @flow */
import { GitHub, Instagram, Linkedin, Twitter } from 'react-feather'
import React from 'react'
import styled from 'styled-components'

const GITHUB_COLOR = '#333333'
const GITHUB_URL = 'https://github.com/jessestuart/'
const INSTAGRAM_COLOR = '#C74861'
const INSTAGRAM_URL = 'https://instagram.com/jaystu/'
const LINKEDIN_COLOR = '#0077B5'
const LINKEDIN_URL = 'https://www.linkedin.com/in/jessedstuart/'
const TWITTER_COLOR = '#1DA1F2'
const TWITTER_URL = 'https://twitter.com/jesse_stuart/'

type Icon = $ReadOnly<{
  color: string,
  Image: *,
  url: string,
}>

const Icons: Array<Icon> = Object.freeze([
  {
    Image: GitHub,
    url: GITHUB_URL,
    color: GITHUB_COLOR,
  },
  {
    Image: Linkedin,
    url: LINKEDIN_URL,
    color: LINKEDIN_COLOR,
  },
  {
    Image: Instagram,
    url: INSTAGRAM_URL,
    color: INSTAGRAM_COLOR,
  },
  {
    Image: Twitter,
    url: TWITTER_URL,
    color: TWITTER_COLOR,
  },
])

const BlackMirrorDiv = styled.div.attrs({
  className: 'flex justify-around mv4',
})`
  @media (max-width: 45em) {
    display: none !important;
  }
`

const BlackMirror = () => (
  <BlackMirrorDiv>
    {Icons.map(({ color, Image, url }) => (
      <a key={url} href={url} className="ma1">
        <Image
          fill={color}
          strokeWidth="1px"
          className="b--light-gray hvr-grow"
          size="35px"
        />
      </a>
    ))}
  </BlackMirrorDiv>
)

export default BlackMirror
