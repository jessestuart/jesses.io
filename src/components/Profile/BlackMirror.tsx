import React from 'react'

import { GitHub, Instagram, Linkedin, Twitter } from 'react-feather'
import styled from 'styled-components'

const GITHUB_COLOR = '#333333'
const GITHUB_URL = 'https://github.com/jessestuart/'
const INSTAGRAM_COLOR = '#C74861'
const INSTAGRAM_URL = 'https://instagram.com/jaystu/'
const LINKEDIN_COLOR = '#0077B5'
const LINKEDIN_URL = 'https://www.linkedin.com/in/jessedstuart/'
const TWITTER_COLOR = '#1DA1F2'
const TWITTER_URL = 'https://twitter.com/jesse_stuart/'

interface Icon {
  Image: any
  color: string
  url: string
}

const Icons: Icon[] = [
  {
    Image: GitHub,
    color: GITHUB_COLOR,
    url: GITHUB_URL,
  },

  {
    Image: Linkedin,
    color: LINKEDIN_COLOR,
    url: LINKEDIN_URL,
  },

  {
    Image: Instagram,
    color: INSTAGRAM_COLOR,
    url: INSTAGRAM_URL,
  },

  {
    Image: Twitter,
    color: TWITTER_COLOR,
    url: TWITTER_URL,
  },
]

const BlackMirrorDiv = styled.div.attrs({
  className: 'flex justify-around mv4',
})`
  @media (max-width: 45em) {
    display: none !important;
  }
`

const BlackMirror = () => (
  <BlackMirrorDiv>
    {Icons.map(({ color, Image, url }) => (
      <a key={url} href={url} className="ma1">
        <Image
          fill={color}
          strokeWidth="1px"
          className="b--light-gray hvr-grow"
          size="35px"
        />
      </a>
    ))}
  </BlackMirrorDiv>
)

export default BlackMirror
