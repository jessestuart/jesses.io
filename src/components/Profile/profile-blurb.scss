img.emoji {
  height: 1.5em;
  width: 1.5em;
  margin: 0 0.05em 0 0.1em;
  vertical-align: -0.1em;
}
