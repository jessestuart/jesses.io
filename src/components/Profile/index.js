import React from 'react'
import classNames from 'classnames'

import Avatar from './Avatar'
import ProfileBlurb from './ProfileBlurb'
import ProfileFooter from './ProfileFooter'

import './profile-intro.module.scss'
import styles from './profile-intro.module.scss'

const Profile = () => (
  <div
    className={classNames(
      'items-center justify-center pv4',
      styles.introWrapper
    )}
  >
    <div
      className={classNames('center mw-100-ns mw-50', styles.avatarContainer)}
    >
      <Avatar />
    </div>
    <div className={classNames('tc tr-ns ph2', styles.blurbContainer)}>
      <ProfileBlurb />
    </div>
    <div className={styles.footerContainer}>
      <ProfileFooter />
    </div>
  </div>
)

export default Profile
