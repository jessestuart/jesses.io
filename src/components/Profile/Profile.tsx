import React from 'react'

import styled from 'styled-components'

import Avatar from './Avatar'
import BlackMirror from './BlackMirror'
import ProfileBlurb from './ProfileBlurb'
import ProfileFooter from './ProfileFooter'

const IntroWrapper = styled.div.attrs({
  className: 'items-center justify-center pv4',
})`
  display: grid;
  grid-template-columns: repeat(12, 1fr);
  min-height: calc(100vh - 23px);

  @media (min-width: 45em) {
    grid-template-rows: repeat(2, 1fr);
  }
`

const AvatarContainer = styled.div.attrs({
  className: 'center w-100-ns w-50',
})`
  grid-column: 1 / 13;

  @media (min-width: 45em) and (max-width: 60em) {
    grid-column: 2 / 5;
    grid-row: 1 / 5;
  }
  @media (min-width: 60em) {
    grid-column: 3 / 5;
    grid-row: 1 / 5;
  }
`

const BlurbContainer = styled.div.attrs({
  className: 'tc tr-ns ph2',
})`
  grid-column: 1 / 13;

  @media (min-width: 45em) and (max-width: 60em) {
    grid-column: 5 / 12;
    grid-row: 1 / 5;
  }
  @media (min-width: 60em) {
    grid-column: 5 / 10;
    grid-row: 1 / 5;
  }
`

const FooterContainer = styled.div.attrs({
  className: 'bt b--light-silver',
})`
  grid-column: 1 / 13;
  grid-row: 5 / 7;
`

const Profile = ({ avatar }: any) => (
  <IntroWrapper>
    <AvatarContainer>
      <Avatar image={avatar} />
      <BlackMirror />
    </AvatarContainer>
    <BlurbContainer>
      <ProfileBlurb />
    </BlurbContainer>
    <FooterContainer>
      <ProfileFooter />
    </FooterContainer>
  </IntroWrapper>
)

export default Profile
