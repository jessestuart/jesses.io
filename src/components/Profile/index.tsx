import Avatar from './Avatar'
import BlackMirror from './BlackMirror'
import Profile from './Profile'
import ProfileBlurb from './ProfileBlurb'
import ProfileFooter from './ProfileFooter'
import ProfileHeading from './ProfileHeading'
import ProfileStyledLink from './ProfileStyledLink'

export {
  Avatar,
  BlackMirror,
  Profile,
  ProfileBlurb,
  ProfileFooter,
  ProfileHeading,
  ProfileStyledLink,
}
