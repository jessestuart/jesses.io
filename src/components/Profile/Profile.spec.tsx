import React from 'react'
import { Profile } from './Profile'
import renderer from 'react-test-renderer'

jest.mock('gatsby-image')

describe('Render Profile component.', () => {
  test('Render Profile component.', () => {
    const component = renderer.create(<Profile avatar={{}} />)
    let tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
