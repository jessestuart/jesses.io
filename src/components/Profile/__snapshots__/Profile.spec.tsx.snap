// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Profile component. Render Profile component. 1`] = `
<div
  className="sc-dnqmqq items-center justify-center pv4 dySFDt"
>
  <div
    className="sc-iwsKbI center w-100-ns w-50 hOqydo"
  >
    <div
      className="sc-bdVaJa flex justify-around mv4 cDifEl"
    >
      <a
        className="ma1"
        href="https://github.com/jessestuart/"
      >
        <svg
          className="b--light-gray hvr-grow"
          fill="#333333"
          height="35px"
          stroke="currentColor"
          strokeLinecap="round"
          strokeLinejoin="round"
          strokeWidth="1px"
          viewBox="0 0 24 24"
          width="35px"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M9 19c-5 1.5-5-2.5-7-3m14 6v-3.87a3.37 3.37 0 0 0-.94-2.61c3.14-.35 6.44-1.54 6.44-7A5.44 5.44 0 0 0 20 4.77 5.07 5.07 0 0 0 19.91 1S18.73.65 16 2.48a13.38 13.38 0 0 0-7 0C6.27.65 5.09 1 5.09 1A5.07 5.07 0 0 0 5 4.77a5.44 5.44 0 0 0-1.5 3.78c0 5.42 3.3 6.61 6.44 7A3.37 3.37 0 0 0 9 18.13V22"
          />
        </svg>
      </a>
      <a
        className="ma1"
        href="https://www.linkedin.com/in/jessedstuart/"
      >
        <svg
          className="b--light-gray hvr-grow"
          fill="#0077B5"
          height="35px"
          stroke="currentColor"
          strokeLinecap="round"
          strokeLinejoin="round"
          strokeWidth="1px"
          viewBox="0 0 24 24"
          width="35px"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M16 8a6 6 0 0 1 6 6v7h-4v-7a2 2 0 0 0-2-2 2 2 0 0 0-2 2v7h-4v-7a6 6 0 0 1 6-6z"
          />
          <rect
            height="12"
            width="4"
            x="2"
            y="9"
          />
          <circle
            cx="4"
            cy="4"
            r="2"
          />
        </svg>
      </a>
      <a
        className="ma1"
        href="https://instagram.com/jaystu/"
      >
        <svg
          className="b--light-gray hvr-grow"
          fill="#C74861"
          height="35px"
          stroke="currentColor"
          strokeLinecap="round"
          strokeLinejoin="round"
          strokeWidth="1px"
          viewBox="0 0 24 24"
          width="35px"
          xmlns="http://www.w3.org/2000/svg"
        >
          <rect
            height="20"
            rx="5"
            ry="5"
            width="20"
            x="2"
            y="2"
          />
          <path
            d="M16 11.37A4 4 0 1 1 12.63 8 4 4 0 0 1 16 11.37z"
          />
          <line
            x1="17.5"
            x2="17.5"
            y1="6.5"
            y2="6.5"
          />
        </svg>
      </a>
      <a
        className="ma1"
        href="https://twitter.com/jesse_stuart/"
      >
        <svg
          className="b--light-gray hvr-grow"
          fill="#1DA1F2"
          height="35px"
          stroke="currentColor"
          strokeLinecap="round"
          strokeLinejoin="round"
          strokeWidth="1px"
          viewBox="0 0 24 24"
          width="35px"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M23 3a10.9 10.9 0 0 1-3.14 1.53 4.48 4.48 0 0 0-7.86 3v1A10.66 10.66 0 0 1 3 4s-4 9 5 13a11.64 11.64 0 0 1-7 2c9 5 20 0 20-11.5a4.5 4.5 0 0 0-.08-.83A7.72 7.72 0 0 0 23 3z"
          />
        </svg>
      </a>
    </div>
  </div>
  <div
    className="sc-gZMcBi tc tr-ns ph2 ddEsaJ"
  >
    <h2
      className="f3 fw5 lh-title white-80 spectral"
    >
      Hi — I'm Jesse.
    </h2>
    <h2
      className="f3 fw5 lh-title white-80 mv3 spectral"
    >
      <span
        className="dib nowrap"
      >
        I contradict myself.
      </span>
      <span
        className="dib nowrap ml2"
      >
        I contain 
        <a
          className="dib sc-bwzfXH gekeSr"
          href="http://jstu.art/ojNe"
        >
          multitudes
        </a>
        .
      </span>
    </h2>
    <p
      className="flex justify-center justify-end-ns items-center"
    >
      <span
        className="f4 fw3 dib tr lh-copy"
      >
        NYC-based software engineer.
      </span>
      <span
        className="f6 dib"
      >
        <svg
          className="mh2"
          style={
            Object {
              "height": "1.25rem",
              "width": "1.25rem",
            }
          }
          viewBox="0 0 45 45"
        >
          <defs>
            <clipPath
              id="a"
            >
              <path
                d="M0 36h36V0H0v36z"
              />
            </clipPath>
            <clipPath
              id="b"
            >
              <path
                d="M4 8.912c-1.105 0-1.667-1.004-2-2.008L0 2.009C0 .899.895 0 2 0h32c1.105 0 2 .899 2 2.009l-2 4.895c-.417.963-.895 2.008-2 2.008H4z"
              />
            </clipPath>
            <clipPath
              id="c"
            >
              <path
                d="M0 36h36V0H0v36z"
              />
            </clipPath>
          </defs>
          <g
            clipPath="url(#a)"
            transform="matrix(1.25 0 0 -1.25 0 45)"
          >
            <path
              d="M33 10.92a2.004 2.004 0 0 0-2-2.007H5c-1.104 0-2 .899-2 2.008v17.07C3 29.102 3.896 30 5 30h26c1.105 0 2-.9 2-2.008V10.92zm3-8.912A2.004 2.004 0 0 0 34 0H2C.896 0 0 .9 0 2.008l2 4.895c.333 1.005.896 2.01 2 2.01h28c1.105 0 1.583-1.047 2-2.01l2-4.895z"
              fill="#ccd6dd"
            />
          </g>
          <g
            clipPath="url(#b)"
            transform="matrix(1.25 0 0 -1.25 0 45)"
          >
            <path
              d="M38 1.925H0l1-4.059h37v4.059z"
              fill="#9aaab4"
            />
          </g>
          <g
            clipPath="url(#c)"
            transform="matrix(1.25 0 0 -1.25 0 45)"
          >
            <path
              d="M31 11.925c0-.555-.447-1.004-1-1.004H6c-.552 0-1 .449-1 1.004v15.063c0 .554.448 1.004 1 1.004h24c.553 0 1-.45 1-1.004V11.925z"
              fill="#5dadec"
            />
            <path
              d="M32.906 4.958l-.76 2.175c-.239.46-.635.838-1.187.838H5.11c-.552 0-.906-.408-1.156-1.036l-.688-1.977c-.218-.596.448-1.004 1-1.004h7.578s.937.047 1.103.608c.192.648.415 1.624.463 1.795.074.264.388.531.856.531h8.577c.5 0 .747-.253.811-.566.042-.204.313-1.14.438-1.782.111-.57 1.22-.586 1.22-.586h6.594c.553 0 1.22.471 1 1.004"
              fill="#aebbc1"
            />
            <path
              d="M22.375 2.887h-7.781c-.375 0-.538.343-.485.675.054.331.36 1.792.383 1.963.024.171.275.375.524.375h7.015c.297 0 .49-.163.55-.49.06-.326.302-1.64.321-1.94.02-.301-.168-.583-.527-.583"
              fill="#9aaab4"
            />
          </g>
        </svg>
        <svg
          style={
            Object {
              "height": "1.25rem",
              "width": "1.25rem",
            }
          }
          viewBox="0 0 45 45"
        >
          <defs>
            <clipPath
              id="a"
            >
              <path
                d="M0 36h36V0H0v36z"
              />
            </clipPath>
          </defs>
          <g
            clipPath="url(#a)"
            transform="matrix(1.25 0 0 -1.25 0 45)"
          >
            <path
              d="M1 18.982c0-9.39 7.611-17 17-17 9.39 0 17 7.61 17 17s-7.61 17-17 17c-9.389 0-17-7.61-17-17"
              fill="#ffcb4c"
            />
            <path
              d="M14.524 14.964a.914.914 0 0 0-.312.464.798.798 0 0 0 .589 1.02c4.529 1.023 7.577-1.361 7.706-1.463.384-.306.46-.845.174-1.206-.286-.358-.828-.401-1.211-.096-.111.083-2.524 1.923-6.182 1.097a.91.91 0 0 0-.764.184m.72 9.862c0-1.468-.952-2.656-2.125-2.656s-2.125 1.188-2.125 2.656c0 1.467.952 2.656 2.125 2.656s2.125-1.189 2.125-2.656M26.5 23.764c0-1.468-.95-2.656-2.125-2.656-1.173 0-2.125 1.188-2.125 2.656 0 1.467.952 2.656 2.125 2.656 1.175 0 2.125-1.19 2.125-2.656"
              fill="#65471b"
            />
            <path
              d="M17.276.85s1.265.412 1.43 1.353c.172.97-.625 1.167-.625 1.167s1.041.208 1.172 1.376c.123 1.1-.862 1.363-.862 1.363s.97.4 1.016 1.539c.038.959-.995 1.428-.995 1.428s5.038 1.22 5.555 1.34c.516.12 1.321.616 1.07 1.695-.249 1.08-1.204 1.118-1.698 1.003-.494-.116-6.744-1.567-8.9-2.07-.23-.052-1.308-.301-1.439-.333-.54-.127-.785.11-.405.512.51.536.833 1.129.947 2.113.12 1.036-.232 2.314-.433 2.809-.374.92-1.005 1.649-1.734 1.899-1.138.39-1.946-.321-1.542-1.561.604-1.855.21-3.375-.833-4.293-2.449-2.157-3.589-3.695-2.83-6.973.827-3.575 4.377-5.876 7.952-5.048l3.154.682"
              fill="#f19020"
            />
            <path
              d="M9.296 29.65a.917.917 0 0 0-.391.397.8.8 0 0 0 .393 1.113c4.266 1.83 7.699.043 7.843-.034.433-.23.608-.748.39-1.154-.215-.405-.74-.546-1.172-.318-.123.064-2.832 1.432-6.278-.047a.915.915 0 0 0-.785.042M21.43 25.9a.924.924 0 0 0-.361.423.8.8 0 0 0 .467 1.084c4.38 1.536 7.685-.48 7.824-.567.415-.26.554-.787.31-1.178-.241-.389-.775-.494-1.19-.238-.12.073-2.728 1.621-6.267.38a.916.916 0 0 0-.782.095"
              fill="#65471b"
            />
          </g>
        </svg>
      </span>
    </p>
    <p
      className="flex justify-center justify-end-ns items-center"
    >
      <span
        className="f4 fw3 dib tr lh-copy"
      >
        <span
          className="db"
        >
          Published
           
          <a
            className="sc-bwzfXH gekeSr"
            href="http://jstu.art/oijk"
          >
            NLP researcher
          </a>
          , and
        </span>
        <a
          className="sc-bwzfXH gekeSr"
          href="http://jstu.art/oj1g"
        >
          USPO Patent holder
        </a>
        .
      </span>
      <span
        className="f6 dib"
      >
        <svg
          className="mh2"
          style={
            Object {
              "height": "1.25rem",
              "width": "1.25rem",
            }
          }
          viewBox="0 0 45 45"
        >
          <defs>
            <clipPath
              id="a"
            >
              <path
                d="M0 36h36V0H0v36z"
              />
            </clipPath>
          </defs>
          <g
            clipPath="url(#a)"
            transform="matrix(1.25 0 0 -1.25 0 45)"
          >
            <path
              d="M32 29H4a4 4 0 0 1-4-4V10a4 4 0 0 1 4-4h11.416c.52-.596 1.477-1 2.584-1s2.064.404 2.584 1H32a4 4 0 0 1 4 4v15a4 4 0 0 1-4 4"
              fill="#269"
            />
            <path
              d="M20 9a2 2 0 0 0-4 0v18a2 2 0 0 0 4 0V9z"
              fill="#292f33"
            />
            <path
              d="M18 10a2 2 0 0 0-2-2H4a2 2 0 0 0-2 2v18a2 2 0 0 0 2 2h12a2 2 0 0 0 2-2V10z"
              fill="#99aab5"
            />
            <path
              d="M18 10c-.999 1.998-3.657 2-4 2-2 0-5-2-8-2-1 0-2 .896-2 2v16c0 1.104 1 2 2 2 3.255 0 6 2 8 2 3 0 4-1.896 4-3V10z"
              fill="#e1e8ed"
            />
            <path
              d="M34 10a2 2 0 0 0-2-2H20a2 2 0 0 0-2 2v18a2 2 0 0 0 2 2h12a2 2 0 0 0 2-2V10z"
              fill="#99aab5"
            />
            <path
              d="M18 10c.999 1.998 3.657 2 4 2 2 0 5-2 8-2 1 0 2 .896 2 2v16c0 1.104-1 2-2 2-3.256 0-6 2-8 2-3 0-4-1.896-4-3V10z"
              fill="#ccd6dd"
            />
          </g>
        </svg>
        <svg
          style={
            Object {
              "height": "1.25rem",
              "width": "1.25rem",
            }
          }
          viewBox="0 0 45 45"
        >
          <defs>
            <clipPath
              id="a"
            >
              <path
                d="M0 36h36V0H0v36z"
              />
            </clipPath>
          </defs>
          <g
            clipPath="url(#a)"
            transform="matrix(1.25 0 0 -1.25 0 45)"
          >
            <path
              d="M32 36H10a4 4 0 0 1-4-4V8H4a4 4 0 0 1 0-8h24a4 4 0 0 1 4 4v24a4 4 0 0 1 0 8"
              fill="#ffd983"
            />
            <path
              d="M8 26h24v2H10l-2 1v-3z"
              fill="#e39f3d"
            />
            <path
              d="M10 36a4 4 0 0 1-4-4V7.445A3.955 3.955 0 0 1 4 8a4 4 0 1 1 4-4v24.555A3.955 3.955 0 0 1 10 28a4 4 0 0 1 0 8"
              fill="#ffe8b6"
            />
            <path
              d="M12 32a2 2 0 1 0-4.001.001A2 2 0 0 0 12 32M6 4a2 2 0 1 0-4.001.001A2 2 0 0 0 6 4m24 17a1 1 0 0 0-1-1H11a1 1 0 0 0 0 2h18a1 1 0 0 0 1-1m0-4a1 1 0 0 0-1-1H11a1 1 0 1 0 0 2h18a1 1 0 0 0 1-1m0-4a1 1 0 0 0-1-1H11a1 1 0 1 0 0 2h18a1 1 0 0 0 1-1m0-4a1 1 0 0 0-1-1H11a1 1 0 1 0 0 2h18a1 1 0 0 0 1-1"
              fill="#c1694f"
            />
          </g>
        </svg>
      </span>
    </p>
    <p
      className="flex justify-center justify-end-ns items-center"
    >
      <span
        className="f4 fw3 dib tr lh-copy"
      >
        Neuroscientist by degree.
      </span>
      <span
        className="f6 dib"
      >
        <svg
          className="mh2"
          style={
            Object {
              "height": "1.25rem",
              "width": "1.25rem",
            }
          }
          viewBox="0 0 45 45"
        >
          <defs>
            <clipPath
              id="a"
            >
              <path
                d="M0 36h36V0H0v36z"
              />
            </clipPath>
          </defs>
          <g
            clipPath="url(#a)"
            transform="matrix(1.25 0 0 -1.25 0 45)"
          >
            <path
              d="M30 4h-2c0 3.314-2.63 6-5.875 6-3.244 0-5.875-2.686-5.875-6H10a2 2 0 0 1-4 0 2 2 0 0 1 0-4h24a2 2 0 0 1 0 4"
              fill="#f18f26"
            />
            <path
              d="M20 26v-4a7 7 0 1 0 0-14h-8.485C13.532 5.557 16.584 4 20 4c6.075 0 11 4.925 11 11s-4.925 11-11 11m8.041.929a2 2 0 0 0-2.828 2.828l2.121 2.12a2.001 2.001 0 0 0 2.83.002 2.003 2.003 0 0 0-.002-2.83l-2.121-2.12z"
              fill="#f18f26"
            />
            <path
              d="M33.7 28.343a2.002 2.002 0 0 0-2.83 0l-4.243 4.243a1.999 1.999 0 1 0 2.828 2.829l4.244-4.243a2 2 0 0 0 0-2.83M16.727 12.787a1.002 1.002 0 0 0-1.414 0L11.07 17.03a1.004 1.004 0 0 0 0 1.414l12.021 12.021a1.004 1.004 0 0 0 1.414 0l4.243-4.243a1 1 0 0 0 0-1.414l-12.02-12.02z"
              fill="#fdcb58"
            />
            <path
              d="M18.142 14.201l-5.657 5.657 1.415 1.414 5.657-5.656-1.415-1.415z"
              fill="#f18f26"
            />
            <path
              d="M16.414 5.586a2 2 0 0 0-2.828 0l-9.9 9.9a2 2 0 0 0 2.83 2.829l9.898-9.901a2 2 0 0 0 0-2.828m-7.464.879A.999.999 0 1 0 7.536 5.05L3.293 9.293a1 1 0 1 0 1.415 1.414L8.95 6.465z"
              fill="#67757f"
            />
          </g>
        </svg>
        <svg
          style={
            Object {
              "height": "1.25rem",
              "width": "1.25rem",
            }
          }
          viewBox="0 0 45 45"
        >
          <defs>
            <clipPath
              id="a"
            >
              <path
                d="M0 36h36V0H0v36z"
              />
            </clipPath>
          </defs>
          <g
            clipPath="url(#a)"
            transform="matrix(1.25 0 0 -1.25 0 45)"
          >
            <path
              d="M30 0H6a1 1 0 1 0 0 2h24c2.542 0 4 1.367 4 3.75C34 7.287 32.357 9 30 9h-7a1 1 0 1 0 0 2h7c3.589 0 6-2.715 6-5.25C36 2.257 33.645 0 30 0"
              fill="#ea596e"
            />
            <path
              d="M12 11.5c0 3.037 3.605 5.937 1 7.5-3.407 2.044-6.879-4.293-9-5-3-1-4-3.407-4-5 0-1.838 1.986-1.725 5-2 1.906-.175 5.594-.125 9-1 2.942-.756-2 2.462-2 5.5"
              fill="#66757f"
            />
            <path
              d="M33 12.5C33 21 29.247 25 24 25c-10 0-14-7-14-14 0-5.247 6.753-6 12-6s11 2.253 11 7.5"
              fill="#66757f"
            />
            <path
              d="M9.781 18.156c0 2.594-2.375 3.22-1.187 5.031.706 1.078 2.587.635 3.051.582-.037-1.332 1.605-1.876 2.054-1.05.378-.37.71-.83.962-1.36.914-1.924-.46-4.49-1.958-5.2-1.496-.712-2.922-.133-2.922 1.997"
              fill="#66757f"
            />
            <path
              d="M13.633 19.86c.084-1.279-.48-2.67-1.396-2.915-.914-.246-1.277.727-1.608 1.961-.33 1.234-.95 2.686-.036 2.932.914.246 2.931-.317 3.04-1.978"
              fill="#e6aaaa"
            />
            <path
              d="M9 13a1 1 0 1 0-2 0 1 1 0 0 0 2 0"
              fill="#292f33"
            />
            <path
              d="M10.594 17.688c-1.26 1.717-2.834 1.977-4.49 1.979-.73 0-.687-.48-.104-.543 1.076-.115 1.963-.405 2.077-.666.076-.175-.702.041-1.431.041-1.167 0-1.041-.297-.167-.52 1.875-.479 1.834-1.22 2.896-1.188 1.063.033 1.219.896 1.219.896m15.848 6.988c-1.65.616-3.942 2.241-6.192 1.991-.75-.083-.462-.518 0-.625.61-.14 1.437-.704 1.731-.932.281-.219-2.314.557-4.065-.109-.872-.332-.514-.563.031-.5.816.093 1.61-.172 2.641-.797s6.931.57 5.854.972"
              fill="#66757f"
            />
          </g>
        </svg>
      </span>
    </p>
    <p
      className="flex justify-center justify-end-ns items-center"
    >
      <span
        className="f4 fw3 dib tr lh-copy"
      >
        <a
          className="sc-bwzfXH gekeSr"
          href="http://jstu.art/ojVB"
        >
          French translator
        </a>
         
        by night.
      </span>
      <span
        className="f6 dib"
      >
        <svg
          className="mh2"
          style={
            Object {
              "height": "1.25rem",
              "width": "1.25rem",
            }
          }
          viewBox="0 0 45 45"
        >
          <defs>
            <clipPath
              id="a"
            >
              <path
                d="M0 36h36V0H0v36z"
              />
            </clipPath>
          </defs>
          <g
            clipPath="url(#a)"
            transform="matrix(1.25 0 0 -1.25 0 45)"
          >
            <path
              d="M36 9a4 4 0 0 0-4-4h-8v26h8a4 4 0 0 0 4-4V9z"
              fill="#ed2939"
            />
            <path
              d="M4 31a4 4 0 0 1-4-4V9a4 4 0 0 1 4-4h8v26H4z"
              fill="#002495"
            />
            <path
              d="M24 5H12v26h12V5z"
              fill="#eee"
            />
          </g>
        </svg>
        <svg
          style={
            Object {
              "height": "1.25rem",
              "width": "1.25rem",
            }
          }
          viewBox="0 0 45 45"
        >
          <defs>
            <clipPath
              id="a"
            >
              <path
                d="M0 36h36V0H0v36z"
              />
            </clipPath>
          </defs>
          <g
            clipPath="url(#a)"
            transform="matrix(1.25 0 0 -1.25 0 45)"
          >
            <path
              d="M30.311 35.224C32 17 20 4 .776 5.688c8.2-7.717 21.091-7.588 29.107.429 8.018 8.016 8.148 20.909.428 29.107"
              fill="#ccd6dd"
            />
            <path
              d="M30.705 20.085a1.164 1.164 0 0 1 0-1.644 1.162 1.162 0 1 1 0 1.644M14.683 5.705a1.74 1.74 0 0 1 0-2.465 1.742 1.742 0 1 1 0 2.465m13.968 2.147a2.904 2.904 0 0 0-4.108 0 2.902 2.902 0 0 0 0 4.107 2.902 2.902 0 0 0 4.108 0 2.902 2.902 0 0 0 0-4.107"
              fill="#b8c5cd"
            />
          </g>
        </svg>
      </span>
    </p>
  </div>
  <div
    className="sc-gqjmRU bt b--light-silver gUujYY"
  >
    <div
      className="b--hot-pink bb bg-purple bw2 mb1 pb3 pt4"
      style={
        Object {
          "gridColumn": "1 / 13",
        }
      }
    >
      <h4
        className="f4 fira-mono fw4 lh-title tc white-80"
      >
        I build software
        <br />
        on the pull requests
        <br />
        of giants.
      </h4>
      <div
        className="flex justify-center mt2 w-100"
      >
        <div
          className=""
        >
          <svg
            className="mw-5"
            fill="none"
            height="30px"
            stroke="#FF3A5C"
            strokeLinecap="round"
            strokeLinejoin="round"
            strokeWidth="2"
            viewBox="0 0 24 24"
            width="30px"
            xmlns="http://www.w3.org/2000/svg"
          >
            <polyline
              points="6 9 12 15 18 9"
            />
          </svg>
        </div>
      </div>
    </div>
  </div>
</div>
`;
